lib/puma/minissl/context_builder.rb

```diff
@@ -45,8 +45,9 @@ def context
           ctx.ssl_cipher_filter = params['ssl_cipher_filter'] if params['ssl_cipher_filter']
         end
 
-        ctx.no_tlsv1 = true if params['no_tlsv1'] == 'true'
+        ctx.no_tlsv1   = true if params['no_tlsv1']   == 'true'
         ctx.no_tlsv1_1 = true if params['no_tlsv1_1'] == 'true'
+        ctx.no_tlsv1_3 = true if params['no_tlsv1_3'] == 'true'
```

Comment:
At what point to we decide there's a more elegant interface here than just disabling individual TLS minor versions? I seem to remember something was discussed a long ways back...

Reply:
When we drop support for OpenSSL 1.0.2?  1.1.1 has min/max functions, with 1.0.2 you have to supply a list of allowed protocols.

JFYI, TLSv1.2 came out in 2008, TLSv1.3 in 2018.  What we have now will probably work for a few years...